Gemfile

```diff
@@ -34,14 +34,15 @@ if allow_local && File.exist?('../urbanopt-scenario-gem')
   # gem 'urbanopt-scenario', github: 'URBANopt/urbanopt-scenario-gem', branch: 'develop'
   gem 'urbanopt-scenario', path: '../urbanopt-scenario-gem'
 else
-  gem 'urbanopt-scenario', github: 'URBANopt/urbanopt-scenario-gem', branch: 'develop'
+  gem 'urbanopt-scenario', github: 'URBANopt/urbanopt-scenario-gem', branch: 'post_process'
 end
 
+
 if allow_local && File.exists?('../urbanopt-geojson-gem')
   # gem 'openstudio-extension', github: 'NREL/OpenStudio-extension-gem', branch: 'develop'
-  gem 'urbanopt-geojson', path: '../urbanopt-geojson-gem'
+  gem 'urbanopt-geojson', path: '../urbanopt-scenario-gem'
 else
-  gem 'urbanopt-geojson', github: 'urbanopt/urbanopt-geojson-gem', branch: 'method-extractions'
+  gem 'urbanopt-geojson', github: 'urbanopt/urbanopt-geojson-gem', branch: 'example_project_fixes'
```

Comment:
I am going to merge this https://github.com/urbanopt/urbanopt-geojson-gem/pull/21/files